LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Block Lab
```

Comment:
Note that you may want to update the entity name, wasn't sure if "Block Lab" was official and complete or not.